scenes/game_elements/characters/player/player.tscn

```diff
@@ -536,6 +536,7 @@ script = ExtResource("3_j0tly")
 [node name="PlayerSprite" type="AnimatedSprite2D" parent="."]
 unique_name_in_owner = true
 position = Vector2(0, -17)
+scale = Vector2(0.18, 0.18)
```

Comment:
This is a sign that your assets for the player are much larger than what they should!

This leads to [tiny StoryWeaver](https://youtu.be/3znz3-15g8Y?t=801) if you play one of the lore quests, so unfortunately we can't allow it. I can see that your assets come from an asset pack that has very high resolution. I'm afraid that you'll have to reduce the player assets in a graphics application like Gimp / Photoshop / (Libre)Aseprite, etc if you really want to use them. But before, please post what is this asset pack coming from and check which license does it have.